requirements/base.in

```diff
@@ -2,8 +2,7 @@
 -c constraints.txt
 
 # Django/Framework Packages
-django<2,>=1.11
-django-model-utils>=2.3.1
```

Comment:
Why was this removed?  It's a direct dependency, used in `edx_proctoring/models.py`.

Reply:
I moved that in constraints.